modules/@apostrophecms/login/ui/apos/components/TheApostropheLogin.vue

```diff
@@ -1,55 +1,277 @@
 <template>
-  <div class="apos-login">
-    <form @submit.prevent="submit">
-      <h2 v-if="error">{{ error }}</h2>
-      <fieldset>
-        <label>Username</label>
-        <input v-model="username" required />
-      </fieldset>
-      <fieldset>
-        <label>Password</label>
-        <input v-model="password" required type="password" />
-      </fieldset>
-      <fieldset>
-        <button type="submit">Log In</button>
-      </fieldset>
-    </form>
+  <div class="apos-login__loader" v-if="contextLoading">Loading<AposSpinner /></div>
+  <div class="apos-login apos-theme-dark" v-else>
+    <div class="apos-login__overlay"></div>
+    <div class="apos-login__menu-overlay"></div>
+    <AposLoginBackground />
+
+    <div class="apos-login__header">
+      <label
+        class="apos-login__project apos-login__project-env"
+        :class="[`apos-login__project-env--${context.env}`]"
+      >
+        {{ context.env }}
+      </label>
+      <label class="apos-login__project apos-login__project-name">{{ context.name }}</label>
+      <label class="apos-login--error">{{ error }}</label>
+    </div>
+
+    <div class="apos-login__body">
+      <form>
+        <AposSchema
+          :schema="schema"
+          v-model="doc"
+        />
+        <!-- TODO -->
+        <!-- <a href="#" class="apos-login__link">Forgot Password</a> -->
+        <AposButton
+          class="apos-field--short"
+          :busy="busy"
+          :disabled="disabled"
+          type="primary"
+          label="Login"
+          :modifiers="['gradient-on-hover']"
+          @click="submit"
+        />
+      </form>
+    </div>
+
+    <div class="apos-login__footer">
+      <AposLogo class="apos-login__logo"/>
+      <label class="apos-login__logo-name">ApostropheCMS</label>
+      <label class="apos-login__project-version">Version {{ context.version }}</label>
+    </div>
   </div>
 </template>
 
 <script>
-
 export default {
   name: 'TheApostropheLogin',
   data() {
     return {
-      username: '',
-      password: '',
-      error: false
+      error: '',
+      busy: false,
+      doc: {
+        data: {},
+        hasErrors: false
+      },
+      schema: [
+        {
+          name: 'username',
+          label: 'Username',
+          placeholder: 'Enter username',
+          type: 'string',
+          required: true
+        },
+        {
+          name: 'password',
+          label: 'Password',
+          placeholder: 'Enter password',
+          type: 'string',
+          required: true
+        }
+      ],
+      context: {},
+      contextLoading: true
```

Comment:
I put this to have a loading screen before displaying the actual login screen. Now, there are infos to get from http://localhost:3000/api/v1/@apostrophecms/login/context (environment, version, project name). So it takes a few ms.

If I put `busy: true`, the login screen is still displayed (maybe a spinner will appear in the future), but the view is re-rendered when the `context` info arrive.

And I find this re-rendering ugly:
![login2](https://user-images.githubusercontent.com/11479686/89061783-9dc03b00-d365-11ea-96f9-a06940b66997.gif)

Here is my suggestion (but idk if there is a design from @stuartromanek that could be better):
![login3](https://user-images.githubusercontent.com/11479686/89062108-26d77200-d366-11ea-8d81-3c5a54760038.gif)

Reply:
What is wrong with using `TheApostropheBusy` in the normal way? If it is not finished then it should get finished, we should not reinvent it here.

Reply:
I am saying that `busy: true` should be passed to apos.http.get. There should be no need for another busy screen implementation.